tests/test_telegram.py

```diff
@@ -0,0 +1,130 @@
+from fastapi import status
+import pytest
+
+from app.telegram.handlers import MessageHandler, reply_unknown_user
+from app.telegram.models import Bot, Chat
+
+
+def gen_message(text):
+    return {
+        'update_id': 10000000,
+        'message': {
+            'message_id': 2434,
+            'from': {
+                'id': 666666,
+                'is_bot': False,
+                'first_name': 'Moshe',
+                'username': 'banana',
+                'language_code': 'en'
+            },
+            'chat': {
+                'id': 666666,
+                'first_name': 'Moshe',
+                'username': 'banana',
+                'type': 'private'
+            },
+            'date': 1611240725,
+            'text': f'{text}'
+        }
+    }
+
+
+def test_chat_model():
+    chat = Chat(gen_message('Cool message'))
+    assert chat.message == 'Cool message'
+    assert chat.user_id == 666666
+    assert chat.username == 'banana'
+    assert chat.first_name == 'Moshe'
+
+
+def test_bot_model():
+    bot = Bot("fake bot id", "https://google.com")
+    assert bot.base == 'https://api.telegram.org/botfake bot id/'
+    assert bot.webhook_setter_url == 'https://api.telegram.org/botfake \
+bot id/setWebhook?url=https://google.com/telegram/'
+
+    assert bot.base == bot._set_base_url("fake bot id")
+    assert bot.webhook_setter_url == bot._set_webhook_setter_url(
+        "https://google.com")
+
+    set_request = bot.set_webhook()
+    assert set_request.status_code == 404
```

Comment:
Could use starlette.status for code status

Reply:
👍